packages/server/src/fhir/operations/projectclone.ts

```diff
@@ -32,7 +32,10 @@ class ProjectCloner {
   constructor(
     readonly repo: Repository,
     readonly projectId: string,
-    readonly projectName: string,
+    readonly projectName: string = '',
+    readonly allowedResourceTypes: Array<string> = [],
```

Comment:
Minor:  I think TypeScript's preferred array syntax is now `string[]` rather than `Array<string>`